liberty/EquivCells.cc

```diff
@@ -525,4 +526,11 @@ equivCellTimingArcSets(const LibertyCell *cell1,
   }
 }
 
+bool
+equivCellFootprints(const LibertyCell *cell1,
+                    const LibertyCell *cell2)
+{
+  return cell1->footprint() == cell2->footprint();
```

Comment:
Comparing pointers is not right - this will only work if the cells are in the same library.  You need to do a string comparison.